frontend/src/modules/Dashboard/Components/GroupCard.tsx

```diff
@@ -66,7 +66,7 @@ export const GroupCard = ({
               fontSize: '1rem',
             }}
             onClick={() => {
-              history.push('/editZing')
+              history.push('/editZing', { params: 'z98mmggpO05931CviaUy' }) // TODO: @shichong replace with real docID
```

Comment:
sure

Reply:
but interestingly you are settings "params" as a state rather than a typical URL param. it should be written as "history.push('editZing?id=z98mmggpO05931CviaUy')" instead